excel/src/main/java/org/apache/metamodel/excel/XlsxSpreadsheetReaderDelegate.java

```diff
@@ -152,10 +160,150 @@ public void close() throws IOException {
                 });
     }
 
+    private ColumnType[] getColumnTypes(final Sheet sheet, final Row row) {
```

Comment:
You are copy/pasting a lot of code into this class from the `DefaultSpreadsheetReaderDelegate` class, that seems like a bad idea to me, why not always use the `DefaultSpreadsheetReaderDelegate` in case you need to detect column types and don't change anything in this class?

Reply:
Done with next commit